pywren_ibm_cloud/job/partitioner.py

```diff
@@ -27,7 +27,7 @@
 CHUNK_THRESHOLD = 128*1024  # 128KB
 
 
-def create_partitions(pywren_config, map_iterdata, chunk_size, chunk_number):
+def create_partitions(pywren_config, internal_storage, map_iterdata, chunk_size, chunk_number):
```

Comment:
@JosepSampe i reference all code changes in this functions. I am not sure if i correct, but seems now it uses internal_storage, while before used backend storage. Is this correct? The issues might be when we use internal_storage configured to Infinispan and storage to COS for example. We clean cache if Infinispan frequently, so if cloud_object is persisted in internal_storage, then those objects will be cleaned as well

Reply:
@gilv No, it is not correct, This change is to avoid creating multiple clients to the same storage backend only in the case where internal storage and objects are both in the same backend.